ci-operator/jobs/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-master-periodics.yaml

```diff
@@ -14,6 +14,13 @@ periodics:
     ci-operator.openshift.io/variant: "4.6"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-ci-redhat-developer-service-binding-operator-master-4.6-validate-master-build
+  reporter_config:
+    slack:
+      channel: '#forum-app-services'
+      job_states_to_report:
+      - failure
+      - error
+      report_template: '@openshift-app-services :volcano: Job *{{.Spec.Job}}* ended with *{{.Status.State}}*. <{{.Status.URL}}|View logs> :volcano:'
```

Comment:
Not sure if `@openshift-app-services` will work as `at` the alias or as plain text. But we will know it soon.

Reply:
Thank you for pointing that out... we will see. If that is a plain text, I'll raise additional PR to fix that.